### DISCUSSION TITLE
Dudas Kata

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/CursoIntroPython#64] [Q&A] [answered]
Hola chicos / chicas, ustedes como están realizando las katas? ¿Lo están haciendo en los mismos documentos que descargas desde GitHub o en un documento aparte?

**Comment by OscarPedroza** [comment, marked as answer]
Hola buenas noches, en mi caso yo estoy haciendo mis propios documentos, pero puede ser de las dos formas, si quieres puedes descargar los archivos del repositorio y trabajar desde ahí, o crear tus documentos.

**Reply by carlos-gmx3** [reply]
Igual, yo incluso ando "tomando nota" de los apuntes en un archivo.ipynb para probar y experimentar con los códigos por mi cuenta

**Comment by MSuarez0** [comment]
la verdad, yo trato muy aparte las katas, en VC code instalado en mi compu, para poder aprender y mejorar el manejo de las herramientas, pero es mas que nada como te acomodes tu.

**Comment by LuisGonzalez86** [comment]
Hola yo descargue las a mi equipo

**Reply by AlanDanielRZ** [reply]
Hola, yo cree mis propios archivos .ipynb en VS Code y voy "copiando" las instrucciones del repositorio a mi propio notebook y seguido realizo cada instrucción del ejercicio en un bloque de código de python.

**Comment by ViktorVizcarra** [comment]
lo que yo hice fue descargar una copia en mi pc, despues baje git hub para escritorio y desde ahi enlace mi la ruta donde puse mi copia a un repositorio en mi github y listo todos los cambios que hago desde visual code en los archivos locales se actualizan en mi reposoritorio github
